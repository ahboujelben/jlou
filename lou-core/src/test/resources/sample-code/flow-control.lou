print "first loop";

for (var i = 0; i < 10; i = i + 1){
    print i;
    if (i == 6) {
        break;
    }
}

print "second loop";

var i = 9;
while (i > 4){
    for (var j = 0; j < 2; j = j + 1){
        print i * 10 + j;
    }
   i = i - 1;
}

if (i == 4) {
    print "end";
}