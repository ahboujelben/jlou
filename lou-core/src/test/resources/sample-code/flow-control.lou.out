first loop
0
1
2
3
4
5
6
second loop
90
91
80
81
70
71
60
61
50
51
end